client/my-sites/themes/theme-showcase.jsx

```diff
@@ -332,7 +333,9 @@ class ThemeShowcase extends Component {
 
 		const olarkIdentity = config( 'olark_chat_identity' );
 		const olarkSystemsGroupId = '239c0f99c53692d81539f76e86910d52';
-		const isEligibleForOlarkChat = ! isLoggedIn && 'en' === locale;
+		const cookies = cookie.parse( document.cookie );
```

Comment:
I think this is failing the E2E tests. My hunch is that the SSR rendering is failing because there's no window or document in this context.
I managed to get the failing test to pass locally by wrapping this in a `typeof window !== 'undefined'` check.